morpho5/docs/meshslice.md

```diff
@@ -0,0 +1,43 @@
+[comment]: # (Meshslice module help)
+[version]: # (0.5)
+
+# Meshslice
+[tagmeshslice]: # (meshslice)
+
+The `meshslice` module is used to slice a `Mesh` object along a given plane, yielding a new `Mesh` object of lower dimensionality. You can also use `meshslice` to project `Field` objects onto the new mesh.
+
+To use the module, begin by importing it:
+
+    import meshslice 
+
+Then construct a `MeshSlicer` object, passing the mesh you want to slice in the constructor:
+
+    var slice = MeshSlicer(mesh)
+
+You then perform a slice by calling the `slice` method, passing the plane you want to slice through. This method returns a new `Mesh` object comprising the slice. A plane is defined by a point that lies on the plane `pt` and a direction normal to the plan `dirn`:
```

Comment:
Do we have need for planes in other places? Is it worth thinking about making a module for a collection of geometry objects (sphere, cube, prism, plane, cylinder)? It would allow users to do things like define a plane with 3 points, or create a mesh from a (possible collection of) geometric objects

Reply:
Great point, this could be a good future module.